### PR TITLE
Saml updated docs

### PR DESCRIPTION
**Pull request by mooreds**
Updated API and samlv2idp docs per https://github.com/FusionAuth/fusionauth-site/issues/375

### CODE REVIEW

**Comment by mooreds** [comment]
@robotdan do you want to take a pass? Would love to ship this today or tomorrow.

File: site/docs/v1/tech/apis/_application-request-body.adoc [hunk 1]
```diff
@@ -286,27 +286,84 @@ When HTTP POST Bindings are used, this is the default verification key used if a
...
+ * `OnlyOriginator` - no other session participants will be notified when a logout request is sent for this application
++
+This configuration is functionally equivalent to the Logout Behavior found in the OAuth2 configuration.
+
+[field]#application.samlv2Configuration.logout.defaultVerificationKeyId# [type]#[UUID]# [optional]#Optional# [since]#Available since 1.25.0#::
+The unique Id of the Key used to verify the signature if the public key cannot be determined by the `KeyInfo` element when using POST bindings, or the key used to verify the signature when using HTTP Redirect bindings.
++
+This parameter is required when [field]#application.samlv2Configuration.logout.requireSignedRequests# is set to `true`. 
+
+[field]#application.samlv2Configuration.logout.keyId# [type]#[UUID]# [optional]#Optional# [default]#Defaults to the key defined by `application.samlv2Configuration.keyId`# [since]#Available since 1.25.0#::
```
Comment (robotdan) [inline site/docs/v1/tech/apis/_application-request-body.adoc:311]: We do have a pattern we use sometimes when the default value needs to be described. This is ok as well, but if you want to look at some other examples look for usages of `[default]#defaults to **[see description]**`.

File: site/docs/v1/tech/apis/_application-request-body.adoc [hunk 2]
```diff
@@ -286,27 +286,84 @@ When HTTP POST Bindings are used, this is the default verification key used if a
...
++
+include::_xmlSignatureC14nMethod-description.adoc[]
++
+include::../shared/_xmlSignatureC14nMethod-values.adoc[]
++
+:signed_entity!:
+
 [field]#application.samlv2Configuration.logoutURL# [type]#[String]# [optional]#Optional# [default]#Defaults to the system logout URL or `/`# [since]#Available since 1.6.0#::
-The URL that the browser is taken to after the user logs out of the SAML service provider. Often service providers need this URL in order to correctly hook up single-logout. **Note** that FusionAuth does not support the SAML single-logout profile because most service providers to not support it properly.
+The URL that the browser is taken to after the user logs out of the SAML service provider. Often service providers need this URL in order to correctly hook up single-logout.
```
Comment (robotdan) [inline site/docs/v1/tech/apis/_application-request-body.adoc:351]: This is also the URL that will be sent the SAML v2 LogoutResponse using the same bindings that were used to initiate the logout request with the IdP. If POST bindings were used, POST bindings will be used here, and so forth.

**Review by robotdan — COMMENTED** [review]
Looks good!

Nice work! There are a lot of moving pieces here and it looks like you nailed it. Themes, APIs, UI screenshots, etc, etc.

**Pull request merged**